### PR TITLE
Add Canto

### PR DESCRIPTION
**Pull request by dogemos**
Add Canto (EVM) chain

### CODE REVIEW

File: canto/chain.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        "low_gas_price": 10000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+        {
+            "denom": "aevmos"
```
Comment (sunnya97) [inline canto/chain.json:28]:
```suggestion
            "denom": "acanto"
```

File: canto/chain.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        "provider": "Plex"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "http://164.90.154.41:26657",
+        "provider": "Canto"
+      }
```
Comment (sunnya97) [inline canto/chain.json:80]: Need https endpoints

File: canto/chain.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "canto",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Canto",
+  "chain_id": "canto_7700-2",
+  "bech32_prefix": "canto",
```
Comment (sunnya97) [inline canto/chain.json:8]: This needs to be registered in the SLIP-173 repo

**Comment by tombeynon** [comment]
Hey @dogemos! The nodes are reporting `canto_7700-1` where the chain data has `canto_7700-2`. Should we change this to the previous chain ID for now?

**Pull request merged**